_posts/2020-05-08-coupling-and-cohesion.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "Better Software Design With Coupling and Cohesion"
+date: 2020-05-08 17:30:00
+categories: ["learning", "software-development"]
+author: rdormer
+---
+
+One of the most fundamental tasks when writing or refactoring software
+of any kind is breaking the problem down into smaller parts. When
+you're first starting out - and even as you continue to gain
+experience - figuring out what those parts should be, and where they
+should live within a codebase can be a daunting task.  Design patterns
+and principles can help, but trying to keep them in mind as you design
+and implement solutions can be overwhelming. Thankfully, there's a
```

Comment:
@rdormer Maybe consider breaking this paragraph into two? "Thankfully" sounds like a good point to break this paragraph into two.